src/main/java/de/tum/in/www1/artemis/domain/exam/Exam.java

```diff
@@ -145,6 +146,10 @@ public class Exam extends DomainObject {
     @JsonIgnoreProperties("exam")
     private Set<ExamUser> examUsers = new HashSet<>();
 
+    @OneToOne(mappedBy = "exam", cascade = CascadeType.REMOVE)
```

Comment:
I think this is a bit problematic, because it will be loaded `EAGER` by default, even if you specify it to be loaded `LAZY`, however it might not be necessary to do that in many cases.

Reply:
This relation is removed.